src/views/Article.vue

```diff
@@ -360,33 +371,40 @@ p {
   }
 }
 
-#category {
+.category {
   color: $blue;
   font-size: 18px;
+  margin-bottom: 16px;
 }
 
-#title-text {
+h1.title-text {
   font-size: 36px;
   font-weight: bold;
   line-height: 56px;
+  margin-bottom: 10px;
 }
```

Comment:
Can CSS specificity be reduced?
```suggestion
.title-text {
  font-size: 36px;
  font-weight: bold;
  line-height: 56px;
  margin-bottom: 10px;
}
```